tests/app/test_session.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from flask import url_for
+
+
+@pytest.mark.usefixtures("app")
+@pytest.mark.parametrize(
+    "todo, expected",
+    [
+        (
+            True,
+            "/sign-out?todo=True",
+        ),
+        (
+            False,
+            "/sign-out?todo=False",
```

Comment:
If the param arg is supposed to be case insensitive then maybe worth adding lower/upper/mixed cases

Reply:
I've changed it to case sensitive now as it always seems to be correct case in the template, even if we hardcode it to `todo=true` or `todo=false`.

Reply:
I've changed it now so it's case sensitive as the boolean returned from the template is always correct case, so no need for extra tests.

Reply:
Resolving since it is not redundant.